settings.js

```diff
@@ -2,8 +2,8 @@ let dbname = process.env.DB_NAME || "agenda";
 let dbhost = process.env.DB_HOST || "localhost";
 let dburi = process.env.DB_URI || null;
 let appId = process.env.API_KEY;
-let collection = "agendaJobs";
-let definitions = "jobDefinitions";
+let collection = process.env.COLLECTION || 'agendaJobs';
```

Comment:
Should these be two different environment variables?  Should we also have a way to set this information via the CLI?